pkg/registry/apis/search/register.go

```diff
@@ -73,17 +82,55 @@ func (b *SearchAPIBuilder) GetAPIRoutes() *builder.APIRoutes {
 						},
 					},
 				},
-				Handler: http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-					urlQuery := r.URL.Query().Get("query")
-					searchRequest := &resource.SearchRequest{Query: urlQuery}
+				Handler: func(w http.ResponseWriter, r *http.Request) {
+					// get tenant
+					orgId, err := request2.OrgIDForList(r.Context())
```

Comment:
if i recall the tenant is in cfg as stack_id,  just curious if we can't use that here?

Reply:
FYI -- namespace is the tenant isolation.  Getting org id will work on-prem, but fail in cloud. getting stack will fail on-prem

just use namespace